rskj-core/src/main/java/co/rsk/rpc/netty/JsonRpcWeb3ServerHandler.java

```diff
@@ -48,8 +49,8 @@ public class JsonRpcWeb3ServerHandler extends SimpleChannelInboundHandler<ByteBu
     private final JsonNodeFactory jsonNodeFactory = JsonNodeFactory.instance;
     private final JsonRpcBasicServer jsonRpcServer;
 
-    public JsonRpcWeb3ServerHandler(Web3 service, List<ModuleDescription> filteredModules) {
-        this.jsonRpcServer = new JsonRpcBasicServer(service, service.getClass());
+    public JsonRpcWeb3ServerHandler(Web3 service, List<ModuleDescription> filteredModules, RskSystemProperties rskSystemProperties) {
```

Comment:
As commented in the JsonRpcCustomServer it is not needed to pass the entire RskSystemProperties class.

Reply:
Done